VideoLocker/res/layout/fragment_add_response_or_comment.xml

```diff
@@ -67,13 +61,13 @@
                 android:inputType="text|textMultiLine" />
 
             <FrameLayout
-                style="@style/edX.Widget.CreationButtonLayout"
                 android:id="@+id/btnAddComment"
+                style="@style/edX.Widget.CreationButtonLayout"
```

Comment:
Should the style be defined after the ID? It doesn't have the `android:` namespace, and it could be argued that it's part of the definition of the view. But I suppose it's up to you; you've done it in all the new style definitions here anyway.